### PR TITLE
feat: Add wsl-send-notify for WSL based Linux

### PR DESCRIPTION
**Pull request by TheoD02**
Hello

Added notification functionality for Windows Subsystem for Linux (WSL). Allows notifications to be sent using the [wsl-notify-send](https://github.com/stuartleeks/wsl-notify-send) command, with support for defining title, body.

I implemented it quickly, ready for review :)

Related to this [issue](https://github.com/jolicode/JoliNotif/issues/102)

### CODE REVIEW

File: bin/wsl-notify-send/wsl-notify-send.exe
Comment (lyrixx) [inline bin/wsl-notify-send/wsl-notify-send.exe:1]: Where did you find this file? We cannot accept a binary from someone we don't know very well. Sorry

File: src/Notifier/WSLTerminalNotifier.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Joli\JoliNotif\Notifier;
+
+use Joli\JoliNotif\Notification;
+use JoliCode\PhpOsHelper\OsHelper;
+
+class WSLTerminalNotifier extends CliBasedNotifier implements BinaryProvider
```
Comment (pyrech) [inline src/Notifier/WSLTerminalNotifier.php:17]: I used to name the notifiers according to the exact name of the binary/service (suffixed with `Notifier`). In your case, this should be named  WslSendNotifyNotifier I guess.

**Review by pyrech — COMMENTED** [review]
Could you add some documentation about this new notifier? And also explain where does this notifier comes from (from a security and license point of view)

File: bin/wsl-notify-send/wsl-notify-send.exe
Comment (TheoD02) [reply bin/wsl-notify-send/wsl-notify-send.exe:1]: From the [repository stuartleeks/wsl-notify-send](https://github.com/stuartleeks/wsl-notify-send/releases)

Version: [v0.1.871612270](https://github.com/stuartleeks/wsl-notify-send/releases/tag/v0.1.871612270)

**Comment by pyrech** [comment]
I ensured the binary is the one from the linked project and added a readme and the license from the project. Thanks @TheoD02 for the PR :pray:

**Comment by TheoD02** [comment]
> u added the new WslNotifyS

Hello,

Yes indeed, I don't remember why I put it in Unix 😯 

Need to change that to Windows, actually it's incorrect

**Pull request merged**